sdk/schemaregistry/azure-data-schemaregistry-apacheavro/src/main/java/com/azure/data/schemaregistry/apacheavro/SchemaRegistryAvroException.java

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.schemaregistry.apacheavro;
+
+import com.azure.core.exception.AzureException;
+
+/**
+ * Represents an exception that is thrown when Avro serialization or deserialization fails.
+ */
+public class SchemaRegistryAvroException extends AzureException {
```

Comment:
Since this library is specific to Apache Avro and if we introduce another Avro package, we can't use the same name there. So, we should consider naming this `SchemaRegistryApacheAvroException`

Reply:
That works for me. I think it would be useful to specify it.